src/components/MessageList/index.js

```diff
@@ -1,21 +1,14 @@
-// @ts-check
 import React from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import smoothScrollTo from 'smooth-scroll-to-js';
 
 import AvatarContainer from '../AvatarContainer';
 import MessageContainer from '../MessageContainer';
 import Message from '../Message';
 
 import * as messageActions from '../../actions/messages';
 
-// TODO: We strongly recommend not to do this unless your project strongly needs it.
-window.__forceSmoothScrollPolyfill__ = true;
-
-//import Avatar from '../../themes/default/components/Avatar/index';
-//import networkManager from '../../../es/utils/network';
-import TypingIndicator from '../../themes/default/components/TypingIndicator/index';
```

Comment:
was the TypeIndicator not being used?

Reply:
Nope. There was a bunch of code in here which was commented out, and this happened to also not be referenced anywhere in the file. I'm really not sure what the story is.